speech/unit_tests/test_client.py

```diff
@@ -21,6 +21,26 @@ class TestClient(unittest.TestCase):
     AUDIO_SOURCE_URI = 'gs://sample-bucket/sample-recording.flac'
     AUDIO_CONTENT = '/9j/4QNURXhpZgAASUkq'
 
+    @staticmethod
+    def _make_result(transcript, confidence):
+        from google.cloud.grpc.speech.v1beta1 import cloud_speech_pb2
+        return cloud_speech_pb2.SpeechRecognitionResult(
+            alternatives=[
+                cloud_speech_pb2.SpeechRecognitionAlternative(
+                    transcript=transcript,
+                    confidence=confidence,
```

Comment:
ISTM you could benefit from a unit test that has a result with multiple `alternatives`, no?